### ISSUE TITLE
No exception regarding contrasts for logos if they are interactive (SC 1.4.3, 1.4.11)

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#1275]
When logos are used as labels for links, buttons, form fields, charts, column headings, etc., they should also comply with contrast requirements. This should be included in the Understandings. See also: https://github.com/w3c/wcag/issues/902
